colossalai/quantization/fp8.py

```diff
@@ -11,6 +11,8 @@
 SUPPORT_TORCH_COMPILE = Version(torch.__version__) >= Version("2.4.0")
 SCALE_BYTES = 4
 
+cuda_arch = int("".join(str(i) for i in torch.cuda.get_device_capability()))
```

Comment:
use try-catch block to get better compatibility

Reply:
Fixed